lib/controllers/include/ActuatorController.h

```diff
@@ -33,12 +33,13 @@ class ActuatorController {
   virtual void activateLimitSwitchChecks() final;
   virtual void deactivateLimitSwitchChecks() final;
 
-  virtual bool reportAngleDeg(float &angle) final;
-  virtual bool reportAngularVelocityDegPerSec(float &speed) final;
+  virtual float reportAngleDeg() final;
+  virtual float reportAngularVelocityDegPerSec() final;
 
   /* functions to be overriden by children */
+  /* TODO: Add finer error reporting (encoder read fail vs current sensor read fail, etc) */
```

Comment:
add some point we need to add an error enum for actuator controller failures to differententiate between the different failures. rn we return false in all cases. this will be part of an error reporting task